packages/lambda-tiler/src/index.ts

```diff
@@ -99,4 +100,6 @@ handler.router.get('/v1/tiles/WMTSCapabilities.xml', wmtsCapabilitiesGet);
 
 // Arcgis Vector
 handler.router.get('/v1/arcgis/rest/services/:tileSet/VectorTileServer', arcgisTileServerGet);
+handler.router.post('/v1/arcgis/rest/services/:tileSet/VectorTileServer', OkResponse);
```

Comment:
Did this work with a OkResponse? or shoudl this just respond with the `arcgisTileServerGet`

Reply:
Yes. I tested just working with ok.